observability/akka-http-metrics/src/test/suite/scala/com/daml/metrics/akkahttp/AkkaHttpMetricsSpec.scala

```diff
@@ -22,6 +22,7 @@ import com.daml.metrics.akkahttp.AkkaUtils._
 import com.daml.metrics.api.MetricName
```

Comment:
Please rename to `HttpMetricsInterceptorSpec` as we now have renamed the class under test.